rosidl_runtime_c/include/rosidl_runtime_c/string_functions.h

```diff
@@ -34,6 +34,7 @@ extern "C"
  * All strings must be null-terminated.
  * The rosidl_runtime_c__String structure should be deallocated using the given function
  * rosidl_runtime_c__String__fini() when it is no longer needed.
+ * Call this function with an already initialized string will leak memory.
```

Comment:
```suggestion
 * Calling this function with an already initialized string will leak memory.
```

Fix here and below.

Reply:
fixed https://github.com/ros2/rosidl/pull/472/commits/fe400595300e61ffeef3b156c244270339d7ef63